From: SJ Park <sj@kernel.org>
Date: Wed, 28 Oct 2026 07:45:51 -0700
Subject: [PATCH] ==== various fixes, optimization and cleanups ====

mm/damon: various fixes and cleanup of code, test, and docs

Fix and cleanup various parts of DAMON including its code, test, and
documentats.

Optimize the efficiency and monitoring accuracy of DAMON code with first
five patches.  Remove unnecessary initializations in damon_new_ctx()
with patch 1 from Adrian Huang (Lenovo).  Align MMU notifier ranges when
clearing PTE/PMD young bits with patches 2 and 3 from Shubhang Kaushik
(Ampere) and Aveline Noir.  Avoid DAMON_STAT idle time percentile
overflow on 32b-bit architectures, and optimize out unnecessary folio
LRU isolation from DAMOS_MIGRATE action, with patches 4 and 5 from
Aveline Noir, respectively.

Fix and cleanup tests and documents with remaining four patches.  Let
sysfs.py test does not hang when it is executed under interactive
terminal with patch 6 from Jaeyeong Lee.  Fix typos, grammartical errors
and styles in DAMON_RECLAIM comments and DAMON_LRU_SORT user documents
with patches 7 and 8 from Manuel.  Finally, document how to disable
watermarks of DAMON_RECLAIM with patch 9 from Jinjie Ruan.

Additional notes that are not intended to be in the git history.  This
is a collection of DAMON patches that individually posted by individual
contributors and resposted as an mm.git merge request.

Signed-off-by: SJ Park <sj@kernel.org>
---
 hkml_cv_bogus/hkml_cv_bogus_r55a6ffp | 0
 1 file changed, 0 insertions(+), 0 deletions(-)
 create mode 100644 hkml_cv_bogus/hkml_cv_bogus_r55a6ffp

diff --git a/hkml_cv_bogus/hkml_cv_bogus_r55a6ffp b/hkml_cv_bogus/hkml_cv_bogus_r55a6ffp
new file mode 100644
index 0000000000000..e69de29bb2d1d
-- 
2.47.3

